dm/dm/master/server.go

```diff
@@ -2641,6 +2641,66 @@ func (s *Server) sharedLogic(ctx context.Context, req interface{}, respPointer i
 	return true
 }
 
+func (s *Server) checkStartValidationParams(req *pb.StartValidationRequest) (
+	map[string]map[string]config.SubTaskConfig, string, bool,
+) {
+	explicitModeOrStartTime := req.Mode != nil || req.StartTime != nil
+	mode := config.ValidationFull
+	if req.Mode != nil {
+		mode = req.GetModeValue()
+	}
+	if mode != config.ValidationFull && mode != config.ValidationFast {
```

Comment:
seems only need to check this when `req.Mode != nil`. And also refer ValidationFull in below message rather than hard coded value.

Reply:
https://github.com/pingcap/tiflow/pull/5534/commits/9901a6f85ea1d99b4a5dca180c5811a6e3f9820d